mne/io/kit/tests/test_kit.py

```diff
@@ -185,4 +188,24 @@ def test_hsp_elp():
     assert_array_almost_equal(pts_elp_in_dev, pts_txt_in_dev, decimal=5)
 
 
+def test_decimate():
+    """Test decimation of digitizer headshapes with too many points. """
+    # load headshape and convert to meters
+    hsp = np.loadtxt(sphere_hsp_txt_path)
+    hsp /= 1000
```

Comment:
You can use `_TempDir()` like here https://github.com/mne-tools/mne-python/blob/474b8d2a6e9df06099aff1012a2c0722549ee72d/mne/tests/test_coreg.py#L25 to quickly generate a temporary directory for the test